dashboard/app/models/level_concept_difficulty.rb

```diff
@@ -0,0 +1,35 @@
+# == Schema Information
+#
+# Table name: level_concept_difficulties
+#
+#  id                    :integer          not null, primary key
+#  level_id              :integer
+#  created_at            :datetime         not null
+#  updated_at            :datetime         not null
+#  sequencing            :integer
+#  debugging             :integer
+#  repeat_loops          :integer
+#  repeat_until_while    :integer
+#  for_loops             :integer
+#  events                :integer
+#  variables             :integer
+#  functions             :integer
+#  functions_with_params :integer
+#  conditionals          :integer
```

Comment:
quite rarely. We expect to be wanting to reference this data quite frequently, though, so we chose this storage strategy to optimize for that at the cost of an inflexible schema.